### DISCUSSION TITLE
Can't get iOS app to do anything. Assistance required please...

### DISCUSSION THREAD

**Discussion by damienbutt** [immich-app/immich#753] [Q&A]
Hi all,

I have the docker stack up and running. I am able to log into the web interface etc. No problems on the face of it from that aspect.

Aside from logging in, I can't seem to make the iOS app do anything. I'm not entirely sure what I should be seeing. 

I granted access to all my photos and confirmed that access is granted in the privacy settings. However, I see no photos in the app.  

I can turn the backup on, but there is nothing in the backup selection to select. 

It's as if the app doesn't have permission to view my photos but the permission is definitely granted.

What could I be doing wrong? Must be something simple...

Any help would be greatly appreciated.

Thanks,

D

**Comment by alextran1502** [comment]
@damienbutt I assume you are still on iOS 15 and you have all your photos on iCloud?

**Reply by damienbutt** [reply]
Hi Alex,

I’m on iOS 16.0.2.

Photos are stored in iCloud yes. 

D

**Reply by alextran1502** [reply]
Ah I see, there is a known issue/mechanism previously that all the photos/videos have to be existing locally in order to be seen by the app. 

There are some reports that the issue was solved in iOS16 but apparently not. 

Some context https://github.com/immich-app/immich/issues/477
